gateway/rdb/src/main/kotlin/org/doorip/gateway/rdb/token/RefreshTokenJpaEntity.kt

```
@@ -0,0 +1,34 @@
+@Table(name = "refresh_token")
+@Entity
+internal class RefreshTokenJpaEntity {
+    @Id
+    @Column(name = "refresh_token", columnDefinition = "binary(45)", nullable = false)
+    lateinit var refreshToken: ByteArray
+
+    @Column(name = "user_id", columnDefinition = "bigint", nullable = false)
+    var userId: Long = 0
+
+    @Column(name = "expired_at", columnDefinition = "datetime", nullable = false)
+    lateinit var expiredAt: LocalDateTime
```

detekt (1.23.7)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)